src/org/zanata/jenkins/Notifier.groovy

```diff
@@ -118,18 +125,56 @@ class Notifier implements Serializable {
   void successful(String message='') {
     sendHipChat color: "GRAY", notify: true, message: "SUCCESSFUL: Job " + jobLinkHtml()
     updateGitHubCommitStatus('SUCCESS', 'SUCCESS: ' + message)
+    mailToCommitter(message)
   }
 
   // Used when tests failure, but compile completed
   void failed(String message='') {
     sendHipChat color: "RED", notify: true, message: "FAILED: Job " + jobLinkHtml()
     updateGitHubCommitStatus('FAILURE', 'FAILURE: ' + message)
+    mailToCommitter(message)
   }
 
   // Used when build failure. e.g. build system/script failed, or compile error
   void error(String message='') {
     sendHipChat color: "RED", notify: true, message: "ERROR: Job " + jobLinkHtml()
     updateGitHubCommitStatus('ERROR', 'ERROR: ' + message)
+    mailToCommitter(message)
+  }
+
+  private void mailToCommitter(String message='') {
```

Comment:
@definite 
The method name `mailToCommitter` is bit misleading, and the recipients may change in future. How about calling it simply `sendEmail`?